testsupport/space/spacerequest.go

```diff
@@ -59,6 +66,25 @@ func CreateSpaceRequest(t *testing.T, awaitilities wait.Awaitilities, memberName
 	return spaceRequest, parentSpace
 }
 
+func CreateSpaceRequestForParentSpace(t *testing.T, awaitilities wait.Awaitilities, memberName, parent string, opts ...SpaceRequestOption) *toolchainv1alpha1.SpaceRequest {
+	memberAwait, err := awaitilities.Member(memberName)
+	require.NoError(t, err)
+
+	// wait for the namespace to be provisioned since we will be creating the spacerequest into it.
+	parentSpace, err := awaitilities.Host().WaitForSpace(t, parent, wait.UntilSpaceHasAnyProvisionedNamespaces())
+	require.NoError(t, err)
+
+	// create the space request in the "default" namespace provisioned by the parentSpace
+	namespace := GetDefaultNamespace(parentSpace.Status.ProvisionedNamespaces)
+	spaceRequest := NewSpaceRequest(t, append(opts, WithNamespace(namespace))...)
+	require.NotEmpty(t, spaceRequest)
+
+	// don't cleanup the new spacerequest, since we need it for migration testing
+	err = memberAwait.Create(t, spaceRequest)
+	require.NoError(t, err)
+	return spaceRequest
+}
```

Comment:
could you please move this function to the `setup_runner.go` file? it's very specific to the migration use-case, there is no cleanup activated for the CR, so it's better to keep it there to avoid any confusion

Reply:
Fixed in bdb46f7.